extensions/blocks/podcast-player/style.scss

```diff
@@ -297,6 +293,16 @@ $player-float-background: $light-gray-200;
 		}
 	}
 
+	/**
+	 * Style overrides for AMP version.
+	 */
+	&.is-amp {
+		.jetpack-podcast-player__audio-player {
+			display: block;
+			padding: 0 $player-grid-spacing;
+		}
+	}
```

Comment:
```suggestion
	&.is-amp .jetpack-podcast-player__audio-player {
		display: block;
		padding: 0 $player-grid-spacing;
	}
```

Reply:
I think there will be at least one or two extra rules inside `is-amp` to fix some visual regressions. That's why I made it a wrapper like this.

Reply:
Usually, I'd agree with @obenland here, but in this case this is intended to group all AMP rules. I prefer you set up the precedent for this now so that it's clear that future devs should nest their selectors within this block instead of littering the CSS with `is-amp` conditionals.

Reply:
more rules present now